pytest_fixtures/api_fixtures.py

```diff
@@ -520,3 +520,28 @@ def repo_setup():
     yield details
     for property_name in ['lce', 'repo', 'product', 'org']:
         details[property_name].delete()
+
+
+@pytest.fixture(scope="function")
```

Comment:
Need not to explicitly mention its function scoped fixture.

Just:
```suggestion
@pytest.fixture
```
is enough.

Reply:
Thanks @jyejare fixed.